_kbarticles/2025-03-27-monitoring-app-metrics-opensearch.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Monitoring your application's health using OpenSearch"
+date: March 27, 2025
+excerpt: How to monitor the CPU and memory metrics for your application on Cloud.gov using OpenSearch
+---
+
+One of the benefits of hosting your applications on Cloud.gov is that it offers [performance metrics for
+monitoring the health of your application](https://docs.cloudfoundry.org/loggregator/container-metrics.html).
+
+The most important metrics for monitoring your application's health are the memory and CPU metrics, which can help you identify:
+
+- If your application has enough memory provisioned per instance
+- If your application is experiencing CPU spikes and associated performance issues
+- If you need [to horizontally scale your application instances]({{ site.baseurl }}{% link _kbarticles/2023-08-07-advantages-of-multiple-application-instances.md %})
+
+## A note about CPU metrics
+
+CPUs are virtualized and shared across application containers on a Diego cell virtual machine (VM).
```

Comment:
Provide a little more context on how things run, something like:

Your application runs in one more application containers distributed across Diego cell virtual machines (VMs). CPUs are virtualized and shared with other application containers on the VM.